test/reporter/test-reporter-logger.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
 const test = require('tap').test;
+const proxyquire = require('proxyquire');
 
-const loggerReporter = require('../../lib/reporter/logger');
 const fixtures = require('../fixtures/reporter-fixtures');
 
+const identity = (value) => value;
+const fakeChalk = {
+  yellow: identity
+};
+const loggerReporter = proxyquire('../../lib/reporter/logger', {
+  chalk: fakeChalk
+});
```

Comment:
How come this is required?

Reply:
It's for local runs. Tap users colors now.

Reply:
And this prevents the colors from being printed?

Reply:
It prevents color codes to be added to the output. `lib/reporter/logger.js` uses `chalk.yellow`: https://github.com/nodejs/citgm/blob/2cba5a35084c7e234fb389ca9cb47ea37a8d469e/lib/reporter/logger.js#L10-L13
I'm open to suggestions if there's a better way to handle this.

Reply:
Chalk tests for color support. It can be suppressed with an environment variable and a runtime flag. The problem with using that is that some modules might rely on colours in their tests.

Reply:
I'm just afraid that if we use a global signal like this, `tap` itself will disable colors for the test runner's output.

Reply:
I can try. What would be your suggestion?

Reply:
This can be left as a future improvement, OK?

Reply:
ping @BridgeAR

Reply:
Yes, that's fine for me.